src/worker/main.ts

```diff
@@ -82,8 +82,12 @@ Object.keys(queues).forEach((name) => {
 	});
 
 	queue.on("failed", async (job) => {
-		logger.error({ job, queue: name }, "Job failed");
+		const failedReason = job.failedReason || "failed";
+		logger.error({ job, queue: name, failedReason }, "Job failed");
 		measureElapsedTime(job, { queue: name, status: "failed" });
+
+		const tags = [`queue:${name}`, `failedReason:${failedReason}`];
```

Comment:
Failed reason might contain parameters which potentially explode the cardinality.

We need to remove this tag, it ie enough to have this information in the log only